### ISSUE TITLE
Have you tried to finetune the model on COCO?

### ISSUE THREAD

**Issue by kawaiitakagi** [WeChatCV/WeDetect#10]
Thanks to your extraordinary work on the project. I want to finetune this model on my custom dataset so I ran the model on COCO to see if it's still effective after finetuning. I finally got a result that was AP=0.541 on COCO dataset. Do you think it's normal or I've chosen some inappropriate training settings since training config has not been released so I set some by myself. Here are some major training sets: Frozen image backbone and text backbone, lr=5e-4, weight_decay=0.025, AdamW optimizer and cosine scheduler with 1000 iters warm-up, totally 20 epochs, total batch size=64. Or maybe it's better to wait for your official training config? Thanks a lot again!

**Comment by fushh** [comment]
Sorry for the delayed response. The highest performance we have achieved on the COCO dataset with WeDetect-Base is 56.2 AP. Please refer to our released configuration file for details.

If you get a higher performance, it will be very kind of you to share the config with us.
